AudioManagerCore/src/CAmCommandSender.cpp

```diff
@@ -35,143 +35,151 @@
 #include "CAmDltWrapper.h"
 #include "audiomanagerconfig.h"
 
-#define __METHOD_NAME__ std::string (std::string("CAmCommandSender::") + __func__)
+#define __METHOD_NAME__ std::string(std::string("CAmCommandSender::") + __func__)
 
 namespace am
 {
 
 /**
  *  macro to call all interfaces
  */
-#define CALL_ALL_INTERFACES(...) 														 \
-		std::vector<IAmCommandSend*>::iterator iter = mListInterfaces.begin();	         \
-		std::vector<IAmCommandSend*>::iterator iterEnd = mListInterfaces.end();	         \
-		for (; iter<iterEnd;++iter)													 	 \
-		{																				 \
-			(*iter)->__VA_ARGS__;													 	 \
-		}
-
-CAmCommandSender::CAmCommandSender(const std::vector<std::string>& listOfPluginDirectories, CAmSocketHandler *iSocketHandler) :
-		CAmDatabaseHandlerMap::AmDatabaseObserverCallbacks(),
-        mListInterfaces(),
-        mListLibraryHandles(),
-        mListLibraryNames(),
-        mCommandReceiver(),
-		mSerializer(iSocketHandler)
+#define CALL_ALL_INTERFACES(...)                                             \
+    std::vector<IAmCommandSend *>::iterator iter = mListInterfaces.begin();  \
+    std::vector<IAmCommandSend *>::iterator iterEnd = mListInterfaces.end(); \
+    for (; iter < iterEnd; ++iter)                                           \
+    {                                                                        \
+        (*iter)->__VA_ARGS__;                                                \
+    }
+
+CAmCommandSender::CAmCommandSender(const std::vector<std::string> &listOfPluginDirectories, CAmSocketHandler *iSocketHandler)
+    : CAmDatabaseHandlerMap::AmDatabaseObserverCallbacks()
+    , mListInterfaces()
```

Comment:
The formatting here is a little bit unusual, with the comma at the start of the line instead of the beginning.

(In order to make the size of the diffs managable and not re-review everything I would recommend fixing this in a separate commit)

Reply:
This formatting will stay. I see a lot of benefits with this style:
1. Extending the list is very easy especially for the last line
2. Precompiler #if/#endif will work everywere to exclude or include initializers of the list
3. The initializer list clearly distinguishes from normal body code